lib/IRGen/IRGen.cpp

```diff
@@ -909,7 +913,10 @@ static void performParallelIRGeneration(IRGenOptions &Opts,
     // Create the IR emitter.
     IRGenModule *IGM = new IRGenModule(irgen, std::move(targetMachine),
                                        nextSF, *Context,
-                                       ModuleName, *OutputIter++);
+                                       ModuleName,
+                                       PrimarySpecificPaths(*OutputIter++,
+                                                          nextSF->getFilename(),
+                                       PSPs.SupplementaryOutputs));
```

Comment:
At this point we don't really support any supplementary outputs, so you might as well make an empty set, right?

Reply:
(because otherwise you'd just stomp on the same files over and over)

Reply:
What if -num-threads 1? Useful in that case??

Reply:
I don't think it's important enough to support when we have a perfectly good "don't parallelize" mode. If we ever unify those then it might be more interesting.

Reply:
OK, I will pass in empty supplementary outputs here (by default).
Also the PSPs parameter to the enclosing function becomes unused and so I'm deleting it.
*But*, at the point where performParallelIRGeneration is called, I will add an assertion that the supplementary output paths are empty.